src/worker-thread/index.amp.ts

```diff
@@ -210,31 +206,42 @@ function updateLongTask(global: WorkerGlobalScope) {
 }
 
 /**
- * Walks up a global's prototype chain and dereferences non-whitelisted properties
- * until EventTarget is reached.
  * @param global
  */
 function updateGlobals(global: WorkerGlobalScope) {
-  function deleteUnsafe(object: any, property: string) {
-    if (WHITELISTED_GLOBALS.indexOf(property) >= 0) {
-      return;
-    }
-    try {
-      console.info(`Deleting ${property}...`);
-      delete object[property];
-    } catch (e) {
-      console.warn(e);
+  /**
+   * @param object
+   * @param property
+   * @return True if property was deleted from object. Otherwise, false.
+   */
+  const deleteUnsafe = (object: any, property: string): boolean => {
+    if (WHITELISTED_GLOBALS.indexOf(property) < 0) {
+      try {
+        delete object[property];
+        return true;
+      } catch (e) {
+        console.warn(e);
+      }
     }
-  }
-
+    return false;
+  };
+  // Walk up global's prototype chain and dereference non-whitelisted properties
+  // until EventTarget is reached.
   let current = global;
   while (current && current.constructor !== EventTarget) {
-    console.info('Removing references from:', current);
-    Object.getOwnPropertyNames(current).forEach(prop => deleteUnsafe(current, prop));
+    const deleted: string[] = [];
+    Object.getOwnPropertyNames(current).forEach(prop => {
+      if (deleteUnsafe(current, prop)) {
+        deleted.push(prop);
+      }
+    });
+    console.info(`Removed ${deleted.length} references from`, current, ':', deleted);
     current = Object.getPrototypeOf(current);
   }
-
+  // Wrap global.fetch() with our longTask API.
```

Comment:
Glad to see this comment, perhaps we should make it a bit more generic for future editors?

```suggestion
  // Wrap global methods as necessary
  // 1. global.fetch() with our longTask API.
```